test/parallel/test-console-table.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const { Console } = require('console');
+
+const queue = [];
+
+const console = new Console({ write: (x) => {
+  queue.push(x);
+}, removeListener: () => {} }, process.stderr, false);
+
+function test(data, only, expected) {
+  if (arguments.length === 2) {
+    expected = only;
+    only = undefined;
+  }
+  console.table(data, only);
+  assert.strictEqual(queue.shift(), expected.trimLeft());
+}
+
+common.expectsError(() => console.table([], false), {
+  code: 'ERR_INVALID_ARG_TYPE',
+});
+
+test(null, 'null\n');
+test(undefined, 'undefined\n');
+test(false, 'false\n');
+test('hi', 'hi\n');
+test(Symbol(), 'Symbol()\n');
+
+test([1, 2, 3], `
+┌─────────┬────────┐
+│ (index) │ Values │
+├─────────┼────────┤
+│    0    │   1    │
+│    1    │   2    │
+│    2    │   3    │
+└─────────┴────────┘
+`);
+
+test([Symbol(), 5, [10]], `
+┌─────────┬──────────┐
+│ (index) │  Values  │
+├─────────┼──────────┤
+│    0    │ Symbol() │
+│    1    │    5     │
+│    2    │  [ 10 ]  │
+└─────────┴──────────┘
+`);
+
+test([undefined, 5], `
+┌─────────┬───────────┐
+│ (index) │  Values   │
+├─────────┼───────────┤
+│    0    │ undefined │
+│    1    │     5     │
+└─────────┴───────────┘
+`);
+
+test({ a: 1, b: Symbol(), c: [10] }, `
+┌─────────┬────┬──────────┐
+│ (index) │ 0  │  Values  │
+├─────────┼────┼──────────┤
+│    a    │    │    1     │
+│    b    │    │ Symbol() │
+│    c    │ 10 │          │
+└─────────┴────┴──────────┘
+`);
+
+test(new Map([ ['a', 1], [Symbol(), [2]] ]), `
+┌───────────────────┬──────────┬────────┐
+│ (iteration index) │   Key    │ Values │
```

Comment:
nit: Keys

Reply:
Actually: we might want to change `Values` to `Value` instead. Because it is a single value at the specific index position.